spec/ble-idioms.test.sh

```diff
@@ -1036,15 +1055,5 @@ echo "[${a[@]: -4}][${a[*]: -4}]"
 [z y x][z y x]
 ## END
 
-# Note: Bash behavior depends on the version and the environment.  In some
-# conditions, the index 0x7FFFFFFFFFFFFFFF hits an integer overflow bug.  Then
-# all slicing with a negative index results in empty lists:
-## BUG bash STDOUT:
-[][]
-[][]
-[][]
-[][]
-## END
```

Comment:
Can we leave this part as a code comment? It's still good to keep in a comment what would be the possible output in the Bash with signed integer overflow.